docs/components/aws/elasticache.md

```diff
@@ -0,0 +1,792 @@
+            }]
+        }, function (error, response, redisClient) {
+            if (error) {
+                callback(error);
+            } else {
+                //redisClient.quit();
+            }
+        });
+    }
+});
+```
+### Testing your cluster
+
+**Important:** 
+>**At the moment you cannot test ElasitiCache operations using Sigma IDE test lambda. We will enable this feature
+in near future for you.**
+
+
+### ElastiCache for Redis Operations
+
+Sigma IDE currently supports following categories of Redis operations.
+
+* Key
+* Strings
+* Set 
+
+#### Key Operations
+
+##### 1. DELETE 
+
+- Description
+
+Removes the specified key/keys. A key is ignored if it does not exist.
+
+- Parameters
+
+Parameter | Type   | Required
+:---:     | :---:  | :---:
+Key       | String | :white_check_mark:
+ 
+- Response
+
+Return type | Success | Failure 
+:---:       | :---:   | :---:
+Integer     | `1`     | `0` if key does not exists
+&nbsp;
+
+##### 2. EXPIRE
+
+- Description
+
+Set a timeout in seconds on key. The key will be automatically deleted after the timeout has been expired.
+A key which associated with a timeout is said to be volatile in Redis terminology.
+
+- Parameters
+
+Parameter | Type    | Required
+:---:     | :---:   | :---:
+Key       | String  | :white_check_mark:
+Seconds   | Integer | :white_check_mark:
+ 
+- Response
+
+Return type | Success | Failure 
+:---:       | :---:   | :---:
+Integer     | `1`     | `0` if key does not exists
+&nbsp;
+
+##### 3. PERSIST
+
+- Description
+
+Remove the existing timeout on key, making the key from volatile 
+to persistent (a key with no timeout is association).
+
+- Parameters
+
+Parameter | Type   | Required
+:---:     | :---:  | :---:
+Key       | String | :white_check_mark:
+ 
+- Response
+
+Return type | Success | Failure 
+:---:       | :---:   | :---:
+Integer     | `1`     | `0` if key does not have associated time out or key does not exists
+&nbsp;
+
+##### 4. TYPE
+
+- Description
+
+Remove the existing timeout on key, making the key from volatile 
+to persistent (a key that no timeout is associated).
+
+- Parameters
+
+Parameter | Type   | Required
+:---:     | :---:  | :---:
+Key       | String | :white_check_mark:
+
+- Response
+
+Return type | Success         | Failure 
+:---:       | :---:           | :---:
+String      | Type of the key | `none` if key does not exists
+&nbsp;
+
+##### 5. RENAME
+
+- Description
+
+Rename a key into a new key. If new key already exists then its value will be overwritten.
+
+- Parameters
+
+Parameter | Type   | Required
+:---:     | :---:  | :---:
+Key       | String | :white_check_mark:
+Name      | String | :white_check_mark:
+
+- Response
+
+Return type | Success | Failure 
+:---:       | :---:   | :---:
+String      | `OK`    | An `error` if key does not exists
+&nbsp;
+
+#### Strings Operations
+
+##### 1. APPEND
+
+- Description
+
+Append value at the end of string key if key does exists. If key does not exists,
```

Comment:
Append_s_ _the_ (also make other descriptions singular as such)

exist (* 2)

and append _the_ value